packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol

```diff
@@ -54,7 +54,7 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         uint128 minParticipationRequirement; // Minimum staked tokens that must vote to resolve a request.
         uint128 minAgreementRequirement; // Minimum staked tokens that must agree on an outcome to resolve a request.
         uint128 cumulativeStakeAtRound; // Total staked tokens at the start of the round.
-        uint32 numberOfRequestsToVoteOnInThisRound; // The number of requests that have been voted on in this round.
+        uint32 requestsNumber; // The number of requests that have been voted on in this round.
```

Comment:
`numberOfRequestsToVoteOnInThisRound` seemed, redundant, particularly the "InThisRound" as this is already a variable of  `Round`.

Reply:
This seems bit too generic. Maybe `numberOfRequestsToVote`.

Also the comment should better state "The number of requests that have to be voted on in this round."

Reply:
Fixed in [this commit](https://github.com/UMAprotocol/protocol/pull/4385/commits/fbee9463f95e458044ef7a08c0dc4045790cebef)